server/migrations/20240228164250_postgres_init.sql

```diff
@@ -1,16 +1,18 @@
 CREATE TABLE clients (
-    id TEXT PRIMARY KEY,
+    id SERIAL PRIMARY KEY,
+    serial_number TEXT,
```

Comment:
Should these DB changes be applied via a migration? Or is the schema change big enough here that it warrants us blowing away the db and starting fresh?

Reply:
The change to preserve the DB will include rebuilding the foreign key relationships. Since we don't have use data in production, and I suspect there will be large changes with @dmzmk's sqlx work I think it will be much easier to start over from scratch.